source/mesh_system.c

```diff
@@ -61,10 +65,15 @@ void mesh_system_init(void)
         randLIB_seed_random();
         platform_timer_enable();
         eventOS_scheduler_init();
+#ifndef YOTTA_CFG
+        ns_event_loop_thread_create();
+#endif
         trace_init(); // trace system needs to be initialized right after eventOS_scheduler_init
```

Comment:
(If there was a platform event_init function that eventOS_scheduler_init called, it could maybe have been put there)